search/includes/classes/class-search.php

```diff
@@ -615,4 +630,15 @@ public function filter__epwr_score_mode( $score_mode, $formatted_args, $args ) {
 	public function filter__epwr_boost_mode( $boost_mode, $formatted_args, $args ) {
 		return 'multiply';
 	}
+
+	/*
+	 * Increment the number of queries that have been passed through VIP Search
+	 */
+	private static function query_count_incr() {
+		if ( false === wp_cache_get( self::QUERY_COUNT_CACHE_KEY, self::QUERY_COUNT_CACHE_GROUP ) ) {
```

Comment:
Does `wp_cache_incr()` fail if it doesn't yet exist?

Reply:
Oh we need to do this to set the TTL, got it.

Reply:
Yep.